backend/editor/normalizer.py

```diff
@@ -0,0 +1,30 @@
+import re
```

Comment:
It's ok for now, but as you move to integrate the parser, we will use it as a library, and we'd better have this function there, shared with parsing code.